sound/soc/intel/boards/sof_sdw_max98373.c

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2020 Intel Corporation
+
+/*
+ *  sof_sdw_max98373 - Helpers to handle 2x MAX98373 codec devices
+ *  from generic machine driver
+ */
+
+#include <linux/device.h>
+#include <linux/errno.h>
+#include <sound/soc.h>
+#include <sound/soc-acpi.h>
+#include "sof_sdw_common.h"
+#include "../../codecs/max98373.h"
+
+static const struct snd_soc_dapm_widget mx8373_widgets[] = {
+	SND_SOC_DAPM_SPK("Speaker", NULL),
```

Comment:
I don't know if we should add a prefix here. What happen if both max98373 and rt1308 are used at the same time? @RanderWang

Reply:
In principle we need to add prefix, but is it possible to adopt two different codecs for speaker in single platform ?